lib/validate_params/param_builder.rb

```diff
@@ -43,6 +43,7 @@ def initialize(parent: nil, validations: [])
       end
 
       def param(field, type, options = {})
+        options = ValidateParams::Validatable.configuration.as_json.merge(options)
```

Comment:
The same concern here, it seems to me that calling a `to_h` method instead of `as_json` looks more logical and organic?